doc/pylint.rst

```diff
@@ -1,16 +1,36 @@
-Static code analysis
-====================
+####################
+Static Code Analysis
+####################
 
-The static code analysis tool may be used directly from the :doc:`editor`, or
-by entering manually the Python module or package path - i.e. it works either
-with `.py` (or `.pyw`) Python scripts or with whole Python packages
-(directories containing an `__init__.py` script).
+The **Static Code Analysis** module detects style issues, bad practices, potential bugs, and other quality problems in your code, all without having to actually execute it.
```

Comment:
Please change `module` here to `pane`, to not give users the wrong impression that this is a Python module.

Reply:
Fixed; that was a mistype on my part. I intended to keep to the consistent convention of properly calling each a "pane". Thanks for the catch; I'll formally codify it in the style guide.